getting_started/introduction/first_look_at_the_editor.rst

```diff
@@ -0,0 +1,135 @@
+.. This page is only here to introduce the interface to the user broadly. To
+   cover individual areas in greater detail, write the corresponding pages in
+   the most appropriate section, and link them. E.g. the animation editor goes
+   to the animation section. General pages, for instance, about the project
+   manager, should go in the editor manual.
+
+.. _doc_intro_to_the_editor_interface:
+
+First look at Godot's editor
+============================
+
+This page will give you a brief overview of Godot's interface. We're going to
+look at the different workspaces and docks to help you situate yourself.
+
+.. seealso:: For a comprehensive breakdown of the editor's interface and how to
+             use it, see the :ref:`Editor manual <toc-learn-editor>`.
+
+The Project manager
+-------------------
+
+When you launch Godot, the first window you see is the Project Manager. In the
+default tab, "Projects," you can manage existing projects, import or create new
+ones, and more.
+
+.. image:: img/editor_intro_project_manager.png
+
+At the top of the window, there is another tab named "Templates". You can search
```

Comment:
Note that in 4.0 the layout has changed, see https://github.com/godotengine/godot/pull/41569

Reply:
As 4.0 is still far away and subject to many more changes, including the UI, I'm writing the series for 3.2 so users can benefit from it today and will change it all at once for 4.0 near the release.